bin/check-s3-object.rb

```diff
@@ -102,6 +102,12 @@ class CheckS3Object < Sensu::Plugin::Check::CLI
          long: '--operator-size OPERATION',
          default: 'equal'
 
+  option :no_crit_on_multiple_objects,
+         description: 'If this flag is set, sort all matching objects by last_modified date and check against the newest. By default, this check will return a CRITICAL result if multiple matching objects are found.',
+         short: '-m',
+         long: '--no-crit-on-multiple-objects',
```

Comment:
so as I discovered in https://github.com/sensu-plugins/sensu-plugins-disk-checks/pull/95 the arg parser has special meaning when you have an option that starts with `--no-` which negates it which is not what we want how about `--ok-on-multiple-objects` instead?